src/Project.WebMVC/Areas/Identity/Pages/Account/Manage/Index.cshtml.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Project.ApplicationCore.Entities;
+using Project.Infrastructure.Data;
+
+namespace Project.WebMVC.Areas.Identity.Pages.Account.Manage
+{
+    public class Index : PageModel
+    {
+        private readonly ILogger<Index> _logger;
+        private readonly UserManager<AppUser> _userManager;
+        private readonly SignInManager<AppUser> _signInManager;
+        private readonly AppDbContext _context;
+
+        public Index(
+            ILogger<Index> logger,
+            UserManager<AppUser> userManager, 
+            SignInManager<AppUser> signInManager,
+            AppDbContext context)
+        {
+            _logger = logger;
+            _userManager = userManager;
+            _signInManager = signInManager;
+            _context = context;
+        }
+        
+        public IReadOnlyList<SelectListItem> Languages { get; set; }
+        
+        [BindProperty]
+        public InputModel Input { get; set; }
+        
+        public class InputModel
+        {
+            public string Username { get; set; }
+            
+            [DataType(DataType.PhoneNumber)]
+            public string PhoneNumber { get; set; }
+            
+            public string LanguageId { get; set; }
```

Comment:
Why LanguageId is string? You use int in database, so should be no problems to leave conversion for ModelBinding and use Int. Try later,

Reply:
I tried to use int insted of string and model binding works fine, but select form don't see defualt language. I just don't know how to fix it